content/threads_intro.tex

```diff
@@ -0,0 +1,74 @@
+This section specifies the interaction between \openshmem{} interfaces and the
+user threads, and also describes the routines that can be used for initializing and 
+querying the thread environment. There are four levels of threading supported by
+the \openshmem{} implementation.
+ 
+\begin{itemize}
+\item {\bf \CONST{SHMEM\_THREAD\_SINGLE}} The \openshmem{} program may not be multithreaded, 
+and all \openshmem{} interfaces are invoked by a single thread. 
```

Comment:
I agree. This was an effort to have a parallel structure among thread level description while also emphasizing that the program cannot have multiple threads in the program. 

If you feel strongly about this. We can rephrase to something like this, though it will break the parallel structure between bullets.

> A single thread invokes all openshmem interfaces, and the OpenSHMEM program may not be multithreaded